BetaCameras/ifm/O3D3xx.cs

```diff
@@ -234,6 +234,63 @@ private RangeParamDesc<float> FramerateDesc
         }
         #endregion
 
+        #region Integration Time Mode
+        private O3D3xxIntegrationMode _integrationMode = O3D3xxIntegrationMode.TwoIntegrationTimes;
+        public O3D3xxIntegrationMode IntegrationTimeMode
+        {
+            get
+            {
+                DoEdit((_edit) => {
+                    string[] typeString = _appImager.GetParameter("Type").Split('_');
+                    _integrationMode = (O3D3xxIntegrationMode)EnumUtils.GetEnum(typeof(O3D3xxIntegrationMode), typeString[1]);
+                });
+
+                return _integrationMode;
+            }
+        }
+        private ListParamDesc<O3D3xxIntegrationMode> IntegrationTimeModeDesc
+        {
+            get
+            {
+                ListParamDesc<O3D3xxIntegrationMode> res = new ListParamDesc<O3D3xxIntegrationMode>(typeof(O3D3xxIntegrationMode))
+                {
+                    Description = "Number of different Integration used to process a single frame.",
+                    ReadableWhen = ParamDesc.ConnectionStates.Connected,
+                };
+                return res;
+            }
+        }
+        #endregion
+
+        #region BackgroundDistance
+        private O3D3xxBackgroundDistanceMode _backgroundDistance = O3D3xxBackgroundDistanceMode.MoreThan30;
+        public O3D3xxBackgroundDistanceMode BackgroundDistance
```

Comment:
In the ToF world the term `non-ambiguity range` has been established in order to describe this characteristic. So I would replace `BackgroundDistance` by `NonAmbiguityRange` for all occurences.